packages/components/src/__future__/index.ts

```diff
@@ -1 +1,5 @@
 export * from "./ExampleComponent"
```

Comment:
Should we remove this? Or keep it?

Reply:
Need to keep it for testing purposes or until there's a future component

Reply:
Oh right. Because the exports don't count. Gotcha 👍🏻